### PR TITLE
[ADLS]Create a Package for ADLS

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-files-datalake/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Microsoft
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-files-datalake/LICENSE.txt]: update year

**Comment by zezha-msft** [comment]
The package name should be `azure-storage-file-datalake`, since the File package is `azure-storage-file`.

**Comment by xiafu-msft** [comment]
> The package name should be azure-storage-file-datalake, since the File package is azure-storage-file.

Then should we unify the package name across all sdks?

**Comment by zezha-msft** [comment]
> > The package name should be azure-storage-file-datalake, since the File package is azure-storage-file.
> 
> Then should we unify the package name across all sdks?

Probably, only .NET has "files" instead of "file". But either way we should probably be consistent with the previous Python package.

**Pull request closed without merging**